manifests/2.10.0/opensearch-2.10.0.yml

```diff
@@ -52,7 +52,7 @@ components:
       - gradle:dependencies:opensearch.version
   - name: security
     repository: https://github.com/opensearch-project/security.git
-    ref: '2.10'
+    ref: tags/2.10.0.0
```

Comment:
tags cannot be added before the release. Same for another change

Reply:
We should remove this step from the "pre-release" section of the release instructions then: https://github.com/opensearch-project/security/issues/3028#issue-1810426686. 

The last item reads as updating the tags before the release. Perhaps this is not what is meant?

Reply:
Can you point me where it said tags?

Reply:
I was referring to `Add this repo to the [manifest](https://github.com/opensearch-project/opensearch-build/blob/main/manifests/) for the next developer iteration.` 

I am not sure what that means that is separate from the release branch update--so I figured it meant the tags.

Reply:
Next developer iteration is 2.11.0. So once you bump the version in 2.x branch, add your component to next manifest. The PR is still in review: https://github.com/opensearch-project/opensearch-build/pull/3969

Reply:
Gotcha, my misunderstanding.